src/index.ejs

```diff
@@ -25,23 +25,23 @@
   OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 -->
 <!DOCTYPE html>
-<html lang='en'>
+<html lang="en">
 <head>
   <meta charset="utf-8" />
-  <meta name='theme-color' content='#ff7b1c'>
+  <meta name="theme-color" content="#ff7b1c">
   <meta content="width=device-width, initial-scale=1.0, maximum-scale=1, user-scalable=no" name="viewport"/>
   <meta property="og:type" content="website"/>
   <meta property="og:site_name" content="Squirrel Chat"/>
   <meta property="og:title" content="Squirrel Chat"/>
   <meta property="og:description" content="Next-gen, open-source and enterprise-ready chat platform"/>
-  <meta property="og:image" content="<%= require('@assets/squirrel.png').default %>">
-  <link rel="shortcut icon" href="<%= require('@assets/squirrel.ico').default %>">
-  <link href='https://fonts.googleapis.com/css2?family=Open+Sans:wght@400;600;800&display=swap' rel='stylesheet'/>
+  <meta property="og:image" content="<%= require("@assets/squirrel.png").default %>">
```

Comment:
It'd be preferable to keep single quotes within the tag itself, for the highlighting